plugins/GSdx/GSRenderer.cpp

```diff
@@ -574,11 +574,11 @@ void GSRenderer::KeyEvent(GSKeyEventData* e)
 			return;
 		case VK_DELETE:
 			m_aa1 = !m_aa1;
-			printf("GSdx: (Software) aa1 is now %s.\n", m_aa1 ? "enabled" : "disabled");
+			printf("GSdx: (Software) Edge Anti-aliasing is now %s.\n", m_aa1 ? "enabled" : "disabled");
```

Comment:
"Anti" IMO should be "anti"

You should probably make the Linux and Windows messages the same as well (including whether they are printed to stdout or stderr)